lib/pact_broker/pacts/merger.rb

```diff
@@ -48,7 +48,11 @@ def merge_pacts original_json, additional_json
 
       def same_description_and_state? original, additional
         original["description"] == additional["description"] &&
-          original["provider_state"] == additional["provider_state"]
+          normalized_provider_states(original) == normalized_provider_states(additional)
+      end
+
+      def normalized_provider_states(interaction)
+        interaction.values_at("provider_state", "providerState", "providerStates").compact.first
       end
```

Comment:
Dear god, I've been a Ruby developer for almost 10 years, and I didn't know about the `values_at` method until now. How have I missed this?! Thanks!

Reply:
@bethesque I'm happy that I could make you a better developer :)

Reply:
👏👏 @RaVbaker